### PR TITLE
feat(xlsx renderer) specify ignored columns

### PR DESCRIPTION
**Pull request by jeremielodi**
trying to solve this #3067

### CODE REVIEW

File: test/server-unit/xlsx.js [hunk 1]
```diff
@@ -53,4 +53,21 @@ describe('util.js', () => {
     expect(data.students).to.deep.equal(result);
   });
 
+  it('Should remove unuseful columns for the user such as uuid', () => {
```
Comment (jniles) [inline test/server-unit/xlsx.js:56]: Beautiful test 👍

File: server/lib/renderers/xlsx.js [hunk 2]
```diff
@@ -100,7 +100,17 @@ function render(data, template, options) {
 */
 function find(data, _options) {
   const options = _options || {};
-  return data.rows || data[options.rowsDataKey] || [];
+  const result = data.rows || data[options.rowsDataKey] || [];
+  // check if it's important to remove some unseful columns for the user
+  if (options.ignoredColumns) {
```
Comment (jniles) [inline server/lib/renderers/xlsx.js:109]: This works!

**Review by jniles — APPROVED** [review]
LGTM.

**Comment by jniles** [comment]
@jeremielodi, this is good.  Could you implement a global `IGNORED_COLUMNS` array that always ignores uuids?  You can then combine that with any columns sent from the client.  I'm imagining like this:
```js
const IGNORED_COLUMNS = ['uuid', 'invoice_uuid', 'entity_uuid', 'reference_uuid' /* etc... */];

// inside your find() method
const columnsToIgnore = IGNORED_COLUMNS.concat(ignoredColumns);
rows.forEach(row => {
  // filter out columns, etc ...
});
```

That way, most registries will automatically filter out useless junk.

**Comment by jeremielodi** [comment]
This is a good idea I see but it is important to add a flag saying that uuid's colums should be removed.
Maybe we can at some point need to use the xlsx file for another action

**Comment by jniles** [comment]
Good point - eventually we'll need a way for the user to control it.  But for the moment, let's just aggressively filter them out.

**Review by jniles — APPROVED** [review]
LGTM.

bors r+

**Pull request merged**